fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/ElectionQuestion.kt

```diff
@@ -67,7 +92,7 @@ class ElectionQuestion(electionId: String, question: Question) {
     }
 
     override fun hashCode(): Int {
-      return Objects.hash(title, votingMethod, ballotOptions, writeIn)
+      return Objects.hash(title, votingMethod, ballotOptions)
```

Comment:
Is there a reason why we're excluding the `writeIn` from the hash?

Reply:
I think it was when removing the `writeIn` in argument in `validQuestion` (figuring out that there's noting to check for a boolean here). I probably blindly removed it here thinking it was the `verify().validQuestion` line, since the order of arguments matched...

Nice catch thanks!